npm/test/folderinforetrieval.test.ts

```diff
@@ -0,0 +1,123 @@
+import { TdrFile } from "@nationalarchives/file-information"
+import { String } from "aws-sdk/clients/acm"
+import { UploadForm } from "../src/upload/upload-form"
+
+class MockHTMLForm {
+  html: String = `
```

Comment:
Yes, you are correct. I did this because I thought the HTML getting mutated (when the folder info is added to the HTML) might cause an issue, but what is actually mutated is the HTML that gets written to the Mock "DOM" not this string of HTML, right?